manifests/service.pp

```diff
@@ -3,13 +3,24 @@
 class marathon::service (
   $manage = true,
   $ensure = 'running',
+  $force_provider = undef,
 ) {
   if $manage {
-    service { 'marathon':
-      ensure     => $ensure,
-      enable     => true,
-      hasrestart => true,
-      hasstatus  => true,
+    if $force_provider {
```

Comment:
I don't think this if/else is required. Passing `undef` to `provider` should have the same effect as not passing anything to `provider`.

See what the [deric/mesos](https://github.com/deric/puppet-mesos/blob/master/manifests/service.pp#L37) module does.